src/scratchtocatrobat/converter/converter.py

```diff
@@ -494,7 +494,6 @@ def _create_user_brick(context, scratch_function_header, param_values, declare=F
             user_script_definition_brick_elements_list.add(user_script_definition_brick_element)
             continue
 
-        assert len(function_header_part) == 2
```

Comment:
Why did the addition of "paramid = None " require the removal of this assert? I don't really see the relation between these

Reply:
I removed it because at best it's entirely pointless (because of the assert on the very next line) and at worst it is wrong. The reason I removed it here is because a userscript call had %m.list as a parameter type which was recognized as a list type parameter by Scratch2 but not by Scratch3.